apps/rush-lib/src/cli/logic/pnpm/PnpmLinkManager.ts

```diff
@@ -167,14 +167,19 @@ export class PnpmLinkManager extends BaseLinkManager {
         throw Error(`Dependency "${dependencyName}" is not a symlink in "${pathToLocalInstallation}`);
       }
 
+      // The dependencyLocalInstallationSymlink is just a symlink to another folder.
+      // To reduce the number of filesystem reads that are needed, we will link to where that symlink
+      // it pointed, rather than linking to a link.
+      const dependencyLocalInstallationRealpath: string = fsx.realpathSync(dependencyLocalInstallationSymlink);
```

Comment:
Does this have the potential to break anything if an intermediate target of a symlink chain is changed, but the ultimate target isn't?

Reply:
Shouldn't have any potential issues.